docs/tables/crowdstrike_detection.md

```diff
@@ -0,0 +1,93 @@
+# Table: crowdstrike_detection
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  detection_id,
```

Comment:
Are there other fields we can add here, maybe some high level info about the detection itself?